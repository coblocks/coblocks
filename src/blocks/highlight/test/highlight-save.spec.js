/**
 * External dependencies
 */
import ShallowRenderer from 'react-test-renderer/shallow';

/**
 * Internal dependencies.
 */
import SaveComponent from '../save';

describe( 'coblocks/highlight/save', () => {
	it( 'renders', () => {
		const renderer = new ShallowRenderer();

		const attributes = {
			content: '',
			align: '',
			backgroundColor: '',
			customBackgroundColor: '',
			textColor: '',
			customTextColor: '',
			fontSize: '',
			customFontSize: ''
		};

		renderer.render( <SaveComponent attributes={attributes} /> );

		expect( renderer.getRenderOutput() ).toMatchSnapshot();
	} );
} );
