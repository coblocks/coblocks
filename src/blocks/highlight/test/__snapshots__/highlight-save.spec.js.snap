// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`coblocks/highlight/save renders 1`] = `
<p
  style={
    Object {
      "textAlign": "",
    }
  }
/>
`;
